tests/Feature/Alerts/AlertFilterTest.php

```
@@ -0,0 +1,231 @@
+    $this->admin->assignRole('admin');
+
+    $this->produccion = User::factory()->create(['email_verified_at' => now()]);
+    $this->produccion->assignRole('produccion');
+
+    $this->comercial = User::factory()->create(['email_verified_at' => now()]);
+    $this->comercial->assignRole('comercial');
+
+    $this->rmA = RawMaterial::factory()->create(['code' => 'RM-ALERTA-01']);
+    $this->rmB = RawMaterial::factory()->create(['code' => 'RM-ALERTA-02']);
+
+    $this->alertUnresolvedStockHigh = Alert::create([
```

PHPStan (2.2.8)

[error] Call to an undefined static method App\Models\Alert::create().


(staticMethod.notFound)